src/views/Proposal/Votes.vue

```diff
@@ -90,84 +90,85 @@ watch([sortBy, choiceFilter], () => {
 </script>
 
 <template>
-  <BlockInfiniteScroller
-    :subtree="true"
-    :loading-more="loadingMore"
-    @end-reached="handleEndReached"
-  >
-    <table class="text-left w-full table-fixed">
-      <colgroup>
-        <col class="w-[50%] lg:w-[40%]" />
-        <col class="w-[25%] lg:w-[20%]" />
-        <col class="w-[25%] lg:w-[20%]" />
-        <col class="w-[60px] lg:w-[20%]" />
-        <col class="w-[0px] lg:w-[60px]" />
-      </colgroup>
-      <thead
-        class="bg-skin-bg sticky top-[112px] lg:top-[113px] z-40 after:border-b after:absolute after:w-full"
-      >
-        <tr class="bg-skin-border/10">
-          <th class="pl-4 font-medium">
-            <span class="relative bottom-[1px]">Voter</span>
-          </th>
-          <th class="hidden lg:table-cell">
+  <table class="text-left w-full table-fixed">
```

Comment:
Messy diff because it moved `BlockInfiniteScroller` down the tree.
You can disable whitespace to get cleaner diff:
<img width="575" alt="image" src="https://github.com/snapshot-labs/sx-ui/assets/1968722/ca378892-e697-4091-8f83-449a8d89b4e7">